components/logins/src/schema.rs

```diff
@@ -325,8 +325,8 @@ pub fn upgrade_sqlcipher_db(db: &mut Connection, encryption_key: &str) -> Result
     if user_version < 3 {
         // These indices were added in v3 (apparently)
         tx.execute_all(&[
-            CREATE_OVERRIDE_HOSTNAME_INDEX_SQL,
-            CREATE_DELETED_HOSTNAME_INDEX_SQL,
+            CREATE_OVERRIDE_ORIGIN_INDEX_SQL,
+            CREATE_DELETED_ORIGIN_INDEX_SQL,
         ])?;
```

Comment:
I think these should stay the same.  If a user actually had a v2 database, then they would be using the old column names at this point.

Reply:
Good catch, was a bit over zealous in the renames. Turns out I need to have a new version of that function for the `create` while keeping that one for any v3. Tests pass and migrations work now!